tests/manage/monitoring/conftest.py

```diff
@@ -814,41 +814,42 @@ def delete_target_bucket():
 
 
 @pytest.fixture
-def measure_stop_worker_node(measurement_dir, nodes):
+def measure_stop_worker_nodes(measurement_dir, nodes):
     """
-    Stop one worker node, measure the time when it was stopped and monitors
+    Stop two worker nodes, measure the time when it was stopped and monitors
     alerts that were triggered during this event.
 
     Returns:
         dict: Contains information about `start` and `stop` time for stopping
             worker node
 
     """
-    node = get_nodes(node_type="worker")[0]
+    test_nodes = get_nodes(node_type="worker")[0:2]
 
-    def stop_node():
+    def stop_nodes():
         """
-        Turn off one worker node for 6 minutes.
+        Turn off two worker nodes for 13 minutes.
 
         Returns:
-            str: Node that was turned down
+            list: Names of nodes that was turned down
 
         """
         # run_time of operation
-        run_time = 60 * 6
-        nonlocal node
-        logger.info(f"Turning off node {node.name}")
-        nodes.stop_nodes(nodes=[node])
+        run_time = 60 * 13
+        nonlocal test_nodes
+        node_names = [node.name for node in test_nodes]
+        logger.info(f"Turning off nodes {node_names}")
+        nodes.stop_nodes(nodes=test_nodes)
```

Comment:
Please add a teardown mechanism to bring validate that the nodes are running or start if needed, for cases in which the test fails after this step.
You can use https://github.com/red-hat-storage/ocs-ci/blob/9386b53e5b19689ca0576243f596d8e2c082f23f/ocs_ci/ocs/platform_nodes.py#L383

Reply:
Added finalizer